Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/soup.yml

```
@@ -200,20 +207,21 @@
   - type: SolutionContainerManager
     solutions:
       food:
-        maxVol: 15
+        maxVol: 36 # L5
         reagents:
         - ReagentId: Nutriment
-          Quantity: 6
+          Quantity: 8
         - ReagentId: Vitamin
           Quantity: 4
         - ReagentId: Protein
-          Quantity: 4
+          Quantity: 12
         - ReagentId: CapsaicinOil
-          Quantity: 1
+          Quantity: 8
   - type: Tag
     tags:
       - ClothMade # Mothic Food
       - Soup
+      - Vegetarian # L5
```

There's no `Vegetarian` tag (we just check if there's meat), but we should note the diary.

```suggestion
      - Dairy# L5
```